ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -69,17 +71,28 @@ public void run() {
             .setReuseAddress(true)
             .setReusePort(true);
 
-    final Runner runner =
-        new Runner(
-            config.getChainId().id(),
-            transactionSignerProvider,
-            webClientOptionsFactory.createWebClientOptions(config),
-            applyConfigTlsSettingsTo(serverOptions),
-            downstreamHttpRequestTimeout,
-            jsonDecoder,
-            config.getDataPath());
-
-    runner.start();
+    final Vertx vertx = Vertx.vertx();
+    try {
+      final Context context =
```

Comment:
Does creating the context need to be in the try?

Reply:
it could come out - but everything _after_ creating Vertx _could_ be dangerous - if creating the context fails, we need to close vertx.